City-of-Helsinki/helsinki-paatokset
# PP-272: PP-272: made frontpage calendar items links, fixed policymake…

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-272

Changed frontpage calendar items so that the whole area is a link

Members block is visible if organisation type is not Viranhaltija or Luottamushenkilö

Changed policymaker listings last item to have the members of Kaupunginvaltuusto

**How to test**

Checkout branch and make sure you have the latest meetings in your local environment.

Go to or create a frontpage and check that the whole elements in the upcoming meeting block are now links if a link exists. 

Check that the logic works with the members block. It shouldn't be visible in policymaker pages that are Viranhaltija or Luottamushenkilö

Go or create a page that has /paattajat as its path. The last accordion element should now be "Kaupunginvaltuuston jäsenet" and the members inside are in alphabetical order based on last name.
